modules/packages/TOML.chpl

```diff
@@ -686,13 +710,26 @@ used to recursively hold tables and respective values
       for idx in root.A do this.A[idx] = new unmanaged Toml(root.A[idx]!)?;
       this.tag = root.tag;
     }
-
+   
+    // checks if quotes exist and avoids '.' within them to create subtables
+    proc splitWithoutQuotes(tblPath: string) {
```

Comment:
> [foo."a.b.c".bar]

No, our current implementation doesn't support this. We basically count the number of periods before a quote is encountered and split that many times. I'm not sure how we could handle this case, since we cannot differentiate periods before and after the quotes, using our inbuilt split function.

Reply:
I believe we will need to get rid of using the inbuilt split function and create a new function which would avoid periods inside of quotes.

Reply:
> I believe we will need to get rid of using the inbuilt split function and create a new function which would avoid periods inside of quotes.

I would support this approach. Let's discuss in today's call.